pages/api/getAll.js

```diff
@@ -0,0 +1,15 @@
+import clientPromise from "@/utils/connectMongo";
+
+export default async function handler(req, res) {
```

Comment:
It might be an option to implement the catch-all response to respond with the not implemented (405) for non-implemented request methods.

Reply:
+ it might make sense to check that the request method is GET